site/_data/lib/links.js

```diff
@@ -28,12 +28,13 @@ function getLinkActiveState(itemUrl, pageUrl) {
 }
 
 /**
- * @type {{
- *   sections?: Section[],
+ * @typedef {{
+ *   sections?: SectionLinkItem[],
  *   url?: string,
  *   parent: SectionLinkItem|undefined,
  *   active: boolean,
  * }}
+ * @type {{never}}
```

Comment:
Why never here?

Reply:
I actually made a typo, this is supposed to be `{never}` :)

This means that if you accidentally use SectionLinkItem as a variable (e.g., `const foo = SectionLinkItem`), TSC is more likely to explode. Otherwise the default "type" of the variable is `any`.